api/service/syncing/syncing.go

```diff
@@ -317,7 +318,7 @@ func (sc *SyncConfig) GetBlockHashesConsensusAndCleanUp() bool {
 		Int("maxFirstID", maxFirstID).
 		Int("maxCount", maxCount).
 		Msg("[SYNC] block consensus hashes")
-	if float64(maxCount) >= core.ShardingSchedule.ConsensusRatio()*float64(len(sc.peers)) {
+	if float64(maxCount) >= ConsensusRatio*float64(len(sc.peers)) {
```

Comment:
is this change needed?

Reply:
Right, reverted.  This was for lowering block syncing quorum for localnet test and had nothing to do with VRF/VDF.